samples/hapi-app/webpack.config.js

```diff
@@ -0,0 +1,4 @@
+// Custom Webpack Config that takes control and overrides Archetype
+const { compose, env, options } = require("electrode-archetype-react-app-dev/config/webpack");
+
+module.exports = compose(options);
```

Comment:
show some actual customization example.

Reply:
Am working on simple customization examples, will update the PR once it's done.

Reply:
Added an example to add a plugin while on development environment.